lib/airbrake/rack/notice_builder.rb

```diff
@@ -62,7 +62,7 @@ def add_context(notice)
           context[:action] = @controller.action_name
         end
 
-        notice[:context].merge!(@user.to_hash) if @user
+        notice[:context].merge!(@user.data) if @user
```

Comment:
Technically, this is a serializer. I checked out how people usually name this method and found out that [Django uses `data`](http://www.django-rest-framework.org/api-guide/serializers/). That was convincing, so I decided to rename.